components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link'
```

Comment:
Got me to review the [API reference for Link](https://nextjs.org/docs/api-reference/next/link). Good to know, takeaways… 

- Routing is file-system based
- Default is to push new URLs into client history (with option to replace instead)
- You can [link a dynamic route](https://nextjs.org/docs/api-reference/next/link#if-the-route-has-dynamic-segments) by default

Reply:
yeah, that was good to read too, re: your 2nd bullet for all these Single Page Apps (SPA) things get nuanced quick in terms of performance and a little wisdom and config goes a long way I'm sure.